node/cn/protocol.go

```diff
@@ -134,41 +132,6 @@ type TxPool interface {
 	SubscribeNewTxsEvent(chan<- blockchain.NewTxsEvent) event.Subscription
 }
 
-//go:generate mockgen -destination=node/cn/mocks/blockchain_mock.go -package=mocks github.com/klaytn/klaytn/node/cn BlockChain
```

Comment:
Will you leave "node/cn/mocks/blockchain_mock.go" file?

Reply:
My mistake. I removed it. Thanks!